frontend/packages/dev-console/src/components/topology/components/nodes/OperatorBackedService.scss

```diff
@@ -0,0 +1,33 @@
+.odc-operator-group {
+  fill: #e2e2e2;
+  stroke: #bbbbbb;
+  stroke-width: 4px;
+  stroke-dasharray: 9;
+  cursor: pointer;
```

Comment:
Doesn't appear there is any click nor drag action on service, so there shouldn't be a pointer cursor.